src/toolbar.ts

```diff
@@ -98,6 +98,15 @@ const topbarExtend = {
         ]
     },
     controller: {
+        init() {
+            console.log(this.proxy.getValue('canUndo'))
+            this.proxy.subscribeAppEvent({
```

Comment:
no sure if we want to expose this event, can you please confirm with gunjan sir

Reply:
This event was being used by customer, picked up this from their example